examples/plot_diabetes_variable_importance_example.py

```diff
@@ -54,9 +54,12 @@
 from sklearn.linear_model import LogisticRegressionCV, RidgeCV
 from sklearn.metrics import r2_score, root_mean_squared_error
 from sklearn.model_selection import KFold
+from sklearn.utils import check_random_state
 
 from hidimstat import CPI, LOCO, PFI
 
+seeds = check_random_state(42).randint(1, np.iinfo(np.int32).max, 7)
```

Comment:
Looking at numpy's documentation it seems that using the [`Generator`](https://numpy.org/doc/stable/reference/random/generator.html) class is best practice 
```suggestion
rng = np.random.default_rng(0)
```

That would avoid adding the sklearn.utils import which returns a `RandomState` object. 
The Generator can then be used to generate all the seeds with `random_state=rng.integers(1e3)` for instance.

Reply:
I homogenise the way to define the seed in the example.
Tell me if it's OK for you?

Reply:
Ok, good for me